server/src/tenant/schema-generation/graphql-types/page-info.graphql-type.ts

```diff
@@ -1,18 +1,14 @@
-import {
-  GraphQLBoolean,
-  GraphQLNonNull,
-  GraphQLObjectType,
-  GraphQLString,
-} from 'graphql';
+import { ConnectionCursorScalar } from '@ptc-org/nestjs-query-graphql';
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType } from 'graphql';
 
 /**
  * GraphQL PageInfo type.
  */
 export const PageInfoType = new GraphQLObjectType({
   name: 'PageInfo',
   fields: {
-    startCursor: { type: GraphQLString },
-    endCursor: { type: GraphQLString },
+    startCursor: { type: ConnectionCursorScalar },
```

Comment:
That's also something we need to check. This PageInfo is supposed to live in the /graphql schema but it conflicts with the PageInfo imported from nestjs-query-graphql that should live in /metadata schema. The "hack" we did last week to have 2 different schemas with 2 different endpoints might not work as expected?

Reply:
@Weiko Hmm, we should double check if the `PageInfoType` is only used in module from `/graphql` endpoint to avoid conflit.
If it's the case, I think you're right and something is not working properly